content/blog/2024/02/2024-02-26-jenkins-contributor-summit-and-fosdem-recap.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: "Jenkins Contributor Summit and FOSDEM Recap"
+tags:
+- contributor-summit
+- fosdem
+- events
+- community
+authors:
+- kmartens27
+sig: docs
+description: >
+  Jenkins Contributor Summit and FOSDEM recap and takeaways.
+opengraph:
+  image: /images/post-images/2024/02/26/contributor-summit-group-photo.jpg
+---
+
+The previous in-person Jenkins Contributor Summit took place in 2020, just prior to the lockdowns and precautions that would change the world as a whole.
+However, on February 2, 2024, just prior to this year's FOSDEM conference, we were able to gather in Brussels so that we could have an in-person Jenkins Contributor Summit.
```

Comment:
```suggestion
The previous in-person Jenkins Contributor Summit took place in 2020, just prior to the lockdowns and precautions that would change the world.
Thankfully, on February 2, 2024, just prior to this year's FOSDEM conference, we were able to gather again, in Brussels, so that we could have an in-person Jenkins Contributor Summit.
```